lib/commands/setup.js

```diff
@@ -1391,7 +1392,11 @@ SetupScreens.prototype.sdk = function sdk(callback) {
 			ignoreFiles: new RegExp(this._config.get('cli.ignoreFiles')),
 			validate: function (value) {
 				if (value && (!fs.existsSync(afs.resolvePath(value) || fs.statSync(value).isDirectory()))) {
-					throw new Error(__('Invalid directory'));
+					if (!fs.existsSync(afs.resolvePath(value))) {
+						wrench.mkdirSyncRecursive(value);
```

Comment:
We might need to do a permissions check before we attempt to create a directory. Otherwise, users might end up something like this :

```
Path to find and install Titanium SDKs:
[/Users/pinnamuri/Library/Application Support/Titanium]: /Library/Application Support/Titanium
[ERROR] EACCES, permission denied '/Library/Application Support/Titanium'
```